apiary.apib

```diff
@@ -8,7 +8,46 @@ Docker Runner is a Keboola service, that runs Docker containers as workers in Ke
 
 Encrypts a string or a JSON
 
-## Encryption [/encrypt?componentId={componentId}&projectId={projectId}]
+## Encryption [/encrypt-new?componentId={componentId}&projectId={projectId}&configId={configId}]
```

Comment:
proč je na to novej endpoint? to bude vyžadovat aktualizaci ve všech klientskejch knihovnách. byl bych pro zachování API endpointu, na nic by to nemělo mít vliv.

Reply:
protoze to chci nejdriv nasadit bez zmeny funkce a pak teprv zapnout novy sifrovani, aktualizovat se nikde nic nebude, az to bude chodit, tak v routes.yml prepnem encrypt routu na novej controller a encrypt-new se vyhodi, ale kdyz to zmenim hned, tak nebudu moct otestovat, ze funguje i novy i stary

Reply:
ok, nevěděl jsem, dobrej plán! nemáme ale od toho testing stack?

Reply:
tak mame, ale je to tak velka zmena, ze tomu stejne neverim :)